packages/commerce-sdk-react/src/hooks/ShopperLogin/useShopperLoginHelpers.ts

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2022, Salesforce, Inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-import {ActionResponse, ShopperLoginInstance} from '../types'
-import {ShopperLoginHelpers} from './types'
-
-// The useShopperLoginHelper hook builds on top of the
-// SLAS helpers from commerce-sdk-isomorphic. The actions
-// have one to one relationship with the helpers from
-// commerce-sdk-isomorphic.
-//
-// On top of sending the HTTP response, these helpers will be
-// responsible for managing tokens.
-
-function useShopperLoginHelper(
-    action: ShopperLoginHelpers.LoginGuestUser
-): ActionResponse<ShopperLoginInstance['getAccessToken']>
-function useShopperLoginHelper(
-    action: ShopperLoginHelpers.LoginRegisteredUserB2C
-): ActionResponse<ShopperLoginInstance['getAccessToken']>
-function useShopperLoginHelper(
-    action: ShopperLoginHelpers.RefreshAccessToken
-): ActionResponse<ShopperLoginInstance['getAccessToken']>
-function useShopperLoginHelper(
-    action: ShopperLoginHelpers.Logout
-): ActionResponse<ShopperLoginInstance['logoutCustomer']>
-function useShopperLoginHelper(action: ShopperLoginHelpers): ActionResponse<() => Promise<any>> {
-    // TODO: think about the helper APIs, it doesn't have to follow the action hook signature
-    // since it does more than sending request, it also manages tokens
-    // @ts-ignore TODO: how to declare the type for dynamic key name [action]?
-    return {
-        [action]: () => Promise.resolve(),
-        isLoading: true,
-        error: undefined
-    }
-}
```

Comment:
We still need the SLAS helpers.

Note: The [helpers](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/src/static/helpers/slasHelper.ts) are different than the other actions.

If you take a look at the helper signatures, they look different than the client methods, they takes multiple arguments as oppose to 1 options argument. Which means that the helper can't re-use the `useAsyncExecute` util.

Non-blocking, let's see how we can fix this in the SLAS ticket.

Reply:
Whoops, didn't notice these helpers here. Since they're a special case, I'm happy to leave them for a separate ticket.